assets/router/service-internal.yaml

```diff
@@ -0,0 +1,26 @@
+# Cluster Service with default values
+# Cluster Ingress specific annotations are applied at runtime.
+kind: Service
+apiVersion: v1
+metadata:
+  # Name and Annotations are set at runtime.
+  namespace: openshift-ingress
+  labels:
+    app: router
+spec:
+  type: ClusterIP
+  selector:
+    app: router
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: 80
```

Comment:
If you replace hard-coded ports with named port references (e.g. `targetPort: http`) we can safely change the target container ports without impacting the service definition.

Reply:
Yeah, I did do that change (http and https) -- push was pending based on my local testing. Which I just pushed.